app/src/org/commcare/android/database/connect/models/ConnectJobRecord.java

```diff
@@ -497,4 +514,42 @@ public static ConnectJobRecord fromV4(ConnectJobRecordV4 oldRecord) {
 
         return newRecord;
     }
+
+    public static ConnectJobRecord fromV5(ConnectJobRecordV5 oldRecord) {
+        ConnectJobRecord newRecord = new ConnectJobRecord();
+
+        newRecord.jobId = oldRecord.getJobId();
+        newRecord.title = oldRecord.getTitle();
+        newRecord.description = oldRecord.getDescription();
+        newRecord.status = oldRecord.getStatus();
+        newRecord.completedVisits = oldRecord.getCompletedVisits();
+        newRecord.maxDailyVisits = oldRecord.getMaxDailyVisits();
+        newRecord.maxVisits = oldRecord.getMaxVisits();
+        newRecord.budgetPerVisit = oldRecord.getBudgetPerVisit();
+        newRecord.totalBudget = oldRecord.getTotalBudget();
+        newRecord.projectEndDate = oldRecord.getProjectEndDate();
+        newRecord.lastWorkedDate = oldRecord.getLastWorkedDate();
+        newRecord.deliveries = new ArrayList<>();
+        newRecord.payments = new ArrayList<>();
+        newRecord.learnings = new ArrayList<>();
+        newRecord.assessments = new ArrayList<>();
+        newRecord.paymentUnits = new ArrayList<>();
+
+        newRecord.organization = oldRecord.getOrganization();
+        newRecord.lastWorkedDate = oldRecord.getLastWorkedDate();
+        newRecord.numLearningModules = oldRecord.getNumLearningModules();
+        newRecord.learningModulesCompleted = oldRecord.getLearningModulesCompleted();
+        newRecord.currency = oldRecord.getCurrency();
+        newRecord.paymentAccrued = Integer.toString(oldRecord.getPaymentAccrued());
+        newRecord.shortDescription = oldRecord.getShortDescription();
+        newRecord.lastUpdate = oldRecord.getLastUpdate();
+        newRecord.lastLearnUpdate = oldRecord.getLastLearnUpdate();
+        newRecord.lastDeliveryUpdate = oldRecord.getLastDeliveryUpdate();
+        newRecord.dateClaimed = new Date();
+        newRecord.projectStartDate = new Date();
+        newRecord.isActive = true;
+        newRecord.isUserSuspended=oldRecord.getIsUserSuspended();
```

Comment:
We know the old record won't have any meaningful value since this code is used when upgrading the DB, so just assign false here (i.e. since the old behavior was for all users never to be suspended).

The next time the jobs API call is made, we'll update the DB with fresh data from the server. So this code just handles what to do in the meantime.